iModelCore/iModelPlatform/Tests/DgnProject/NonPublished/VersionCompare_Test.cpp

```diff
@@ -137,7 +137,7 @@ void VersionCompareTestFixture::SetUpTestCase()
     m_db->ResetBriefcaseId(BeBriefcaseId(2));
 
     // Import DgnPlatformTest schema containing the test element and such
-    SchemaStatus schemaStatus = DgnPlatformTestDomain::GetDomain().ImportSchema(*m_db);
+    SchemaStatus schemaStatus = DgnPlatformTestDomain::GetDomain().ImportSchema(*m_db, SchemaManager::SchemaImportOptions::AllowDataTransformDuringSchemaUpgrade);

```

Comment:
seems like you could make `SchemaManager::SchemaImportOptions::AllowDataTransformDuringSchemaUpgrade` the default and then you wouldn't have to change all these tests. The only call that really matters is the one from TS, and it won't use the default.

Reply:
I wanted to be not set by default in ECDb. I guess I will continue that in DgnDb. I have now set it by default.